tests/fixtures/_gen_stubs.py

```diff
@@ -0,0 +1,25 @@
+#!/bin/env python
+
+def tree_paths(tree, prefix=""):
+    if isinstance(tree, dict):
+        return {
+            k: tree_paths(v, "{}__{}__".format(prefix, k))
+            for k, v in tree.items()
+        }
+    elif isinstance(tree, list):
+        return [
+            tree_paths(v, "{}__[{}]__".format(prefix, n))
+            for n, v in enumerate(tree)
+        ]
+    else:
+        return prefix
+
+if __name__ == "__main__":
+    import yaml
+    import sys
+
+    yaml.dump(
+        tree_paths(yaml.load(sys.stdin)),
```

Comment:
Can we double check that we're using the version of `yaml` with safe load - I can't remember if the fix made it here already.

Reply:
Good question!

Reply:
The latest version of PyYAML is pulled in from somewhere, so we could use `safe_load` if we wanted to retain this script.